### PR TITLE
Added /membercount and an auto-updated VC to show member numbers

### PR DESCRIPTION
**Pull request by 49Indium**
The /membercount command displays the current member, count (updated to when the command was called) along with the number of users which joined in the last 7 days.
The VC (which should be visible but not accessible for most users), has its named changed when someone leaves or joins the server, but is capped at refreshing at most every 5 minutes (as the current rate limit is 2 channel name changes per 10 minutes).

### CODE REVIEW

File: uqcsbot/member_counter.py
Comment (andrewj-brown) [inline uqcsbot/member_counter.py:13]: We want to not use IDs as much as possible, because they're opaque to any readers and require funky hacks like this to remain available in other servers (consider: the bot is locally hostable by anyone in any server, not just uqcs & testing). Is it possible to just get all channels and filter by "name starts with Member Count" ?

This is the one case where IDs are justifiable, due to name changes, so don't worry about it too much.

**Review by andrewj-brown — COMMENTED** [review]
I'll defer to other reviewers on whether the IDs are a blocking issue in this PR, but I'd like to see them changed if possible. The name changes make this harder than other use-cases though so the mileage may vary.

File: uqcsbot/member_counter.py
Comment (49Indium) [reply uqcsbot/member_counter.py]: Modified to use the beginning of the channel name to determine the right channel.

On a related note, should `SERVER_ID` within `bot.py` be changed as well? Do we guarantee that uqcsbot is only in one server (as if this is not true, grabbing channels by name is dubious).

Comment (andrewj-brown) [reply uqcsbot/member_counter.py]: uhhhhhhhhhhhhhhhhh good question

Comment (andrewj-brown) [reply uqcsbot/member_counter.py]: alright I may consistent-ise that shortly but for now if you just use the standard approach here (get_all_channels() and filter) then that should be fine

**Review by andrewj-brown — APPROVED** [review]
lgtm

**Comment by 49Indium** [comment]
Given the recent changes, I'll let someone review again. Sorry for the hassle.

**Review by andrewj-brown — APPROVED** [review]
lgtm still. dunno about ending cogs; just blocking further updates should be ok (the only case where it "doesn't work" is an incorrectly configured server, which shouldn't happen in prod)

**Comment by andrewj-brown** [comment]
keep in mind you should setup the channel & perms in the discord server before merging this

**Pull request merged**
